spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+describe UsersController, type: :controller do
+  render_views
+
+  let(:user) { FactoryGirl.create(:user) }
+
+  before do
+    allow(TwitterService.instance).to receive(:screen_name).and_return("foobar")
+  end
+
+  describe "#index" do
+    it "successful" do
+      get :index
+
+      expect(response.status).to eq(200)
+    end
+  end
+
+  describe "#show" do
+    xit "successful" do
```

Comment:
@schmierkov ah, it looks like there's some old stuff. Check http://www.infractoresba.com.ar/leaderboard, you'll see a page which was supposed to be the leaderboard, which is `UsersController#index`, and there are a few broken links to each user, `UsersController#show`, but no associated ERB template. 

I think we can get rid of all user-related controller/routes, as we agreed that we wouldn't have leaderboards, I think...? cc @etagwerker